src/3_PB/SpecificheTecniche/content.typ

```diff
@@ -0,0 +1,253 @@
+
+== Glossario
+Al ﬁne di evitare possibili ambiguità relative al linguaggio utilizzato nei documenti, viene fornito il _Glossario v1.0_, nel quale sono presenti tutte le definizioni di termini aventi uno specifico significato che vuole essere disambiguato. Tali termini, sono scritti in  corsivo e marcati con una G a pedice.
+
+== Riferimenti
+
+=== Riferimenti normativi 
+- Capitolato C6 - InnovaCity: Smart city monitoring platform:
+#link("https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C6.pdf")\
+#link("https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C6p.pdf")
+
+- _Norme di Progetto v1.0_
+- _Analisi dei Requisiti v1.0_
+
+- Regolamento progetto didattico:
+#link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/PD2.pdf")
+
+=== Riferimenti informativi
+
+- Gamma, E., Helm, R., Johnson, R., & Vlissides, J. (1994). *Design Patterns: Elements of Reusable Object-Oriented Software.* Addison-Wesley.
+/*in caso aggiungere qualche riferimento sui dataflow diagram*/
+- Diagrammi delle classi (UML) - corso di Ingegneria del Software a.a. 2023/2024: \
+#link("https://www.math.unipd.it/~rcardin/swea/2023/Diagrammi%20delle%20Classi.pdf") 
+- Diagrammi di Sequenza (UML) - corso di Ingegneria del Software a.a. 2023/2024: \
+#link("https://www.math.unipd.it/~rcardin/swea/2022/Diagrammi%20di%20Sequenza.pdf") /*se non li mettiamo si possono anche togliere*/
+- Progettazione - corso di Ingegneria del Software a.a. 2023/2024: \
+ #link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/T6.pdf") \
+
+=== riferimenti tecnici
+- *#glossary[Python]*: #link("https://docs.python.org/3/");
+- *#glossary[Grafana]*: #link("https://grafana.com/docs/grafana/latest/");
+- *Apache #glossary[Kafka]*: #link("https://kafka.apache.org/20/documentation.html");
+- *#glossary[Clickhouse]*: #link("https://clickhouse.com/docs"); 
+- *Confluent Kafka*:#link("https://docs.confluent.io/kafka-clients/python/current/overview.html");
+
+#pagebreak()
+= Tecnologie
+Questa sezione si propone di offrire una panoramica sintetica delle tecnologie adottate. Questo breve paragrafo introduttivo fornisce un quadro generale delle piattaforme, dei linguaggi di programmazione, dei framework e di altre risorse tecnologiche che costituiscono le basi del nostro progetto. 
+
+== Codifica
+=== Linguaggi e formati dati
+#figure(
+table(
+      columns:(100pt,auto,auto),
+      align: (x, y) => (center, center, center).at(x),
+      fill:(_,row) => if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Tecnologia*],[*Descrizione*],[*Versione*],
+      [#glossary[Python]], [Linguaggio di programmazione ad alto livello noto per la sua semplicità, ampiamente utilizzato nell'analisi dei dati, l'intelligenza artificiale, il calcolo scientifico e altro ancora.], [da definire], 
+      [Sql], [Linguaggio standard per la gestione e la manipolazione dei database], [da definire], 
```

Comment:
Tutti i termini della colonna tecnologie secondo me andrebbero aggiunti al Glossario (con le descrizioni "generiche").

Reply:
Il punto è: serve veramente? Cioè, non c'è nulla di ambiguo. Sono tecnologie.

Reply:
Si ma non è quello il punto, anche Python non è ambiguo ed è in glossario, serve perché se uno non conosce quelle tecnologie se le va a guardare dal glossario e non deve cercarsele in risorse esterne.

Reply:
letteralemnte non credo appariranno in altri documenti oltre a questo, per quello avrei evitato di metterli a glossario. 
(Comunque già qua le descrizioni sono abbastanza generiche credo)